recipes/DABEST/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "dabest" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "12df2715a4fc76ca4abd1fd6a5f2673d8a0e9fa66a8e4797846e1c97c3e904da" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - matplotlib==2.1
```

Comment:
You need a space between the package name and the version spec -- e.g. `matplotlib ==2.1`

Reply:
ok, thanks! fixed!